instance_config.go

```diff
@@ -156,6 +157,16 @@ func (config *InstanceConfig) Validate(registry Registry) error {
 		for _, api := range serverConfig.APIs {
 			presentApis[api.Binding()] = registry.Get(api.Binding())
 		}
+		for _, bp := range serverConfig.BindPoints {
+			ve := config.DefaultIdentity.ValidFor(bp.Address)
```

Comment:
This will only validate against the root identity. A server may have a specific identity to use.

`serverConfig.Identity` should be used. It should be populated w/ DefaultIdentity if an override wasn't supplied. So a simple swap should work.

Reply:
thx. changed and verified the controller will fail as expected